src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/request/UpdateListingRequest.java

```diff
@@ -3,28 +3,43 @@
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+import org.hibernate.validator.constraints.NotEmpty;
 import uk.gov.hmcts.reform.sandl.snlevents.model.Priority;
 import uk.gov.hmcts.reform.sandl.snlevents.model.usertransaction.UserTransactional;
+import uk.gov.hmcts.reform.sandl.snlevents.validation.annotations.MinDuration;
 
 import java.time.Duration;
 import java.time.OffsetDateTime;
 import java.util.UUID;
 
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
 public class UpdateListingRequest implements UserTransactional {
 
+    @NotNull
     private UUID id;
 
+    @NotEmpty
+    @Size(max = 200)
     private String caseNumber;
 
+    @NotEmpty
```

Comment:
should be NotBlank rather than NotEmpty, read the manual. the same in all other places.